app/views/registrations/show.html.erb

```diff
@@ -1,6 +1,8 @@
 <%= tag.h1 'Register DOR Items', class: 'h2 my-4 fw-bolder' %>
 
-<%= form_with model: @registration_form, url: registration_path,
+<%= form_with model: @registration_form,
+              scope: :registration, # added so the form field scope isn't set to `csv_registration`
```

Comment:
remove "added" -- that's only relevant in the context of this PR;  after merge, it won't be "added" any more -- just part of the code.

Reply:
Done